force-app/main/default/lwc/nksFamilyViewerEntry/nksFamilyViewerEntry.js

```diff
@@ -99,35 +138,35 @@ export default class nksFamilyViewerEntry extends NavigationMixin(LightningEleme
         if (this.relation.dateOfDeath != null) {
             return this.relation.dateOfDeath;
         }
-        return 'UKJENT DATE';
+        return this.useNewDesign ? 'Ukjent dato' : 'UKJENT DATE';
     }
 
     get getBirthDate() {
         if (this.relation.birthDate != null) {
             return this.relation.birthDate;
         }
-        return 'UKJENT DATE';
+        return this.useNewDesign ? 'Ukjent dato' : 'UKJENT DATE';
     }
 
     get getSex() {
         if (this.relation.sex != null) {
             return this.relation.sex;
         }
-        return 'UKJENT KJØNN';
+        return this.useNewDesign ? 'Ukjent kjønn' : 'UKJENT KJØNN';
     }
 
     get getChildText() {
         if (this.relation.unauthorized === true || this.relation.deceased) {
             return '';
         }
-        return this.getLiveWithText() + '\n' + this.getResponsibilityChildText();
+        return this.getLiveWithText() + (this.useNewDesign ? '  |  ' : '\n') + this.getResponsibilityChildText();
     }
 
     get getParentText() {
         if (this.relation.unauthorized === true || this.relation.deceased) {
             return '';
         }
-        return this.getLiveWithText() + '\n' + this.getResponsibilityParentText();
+        return this.getLiveWithText() + (this.useNewDesign ? '  |  ' : '\n') + this.getResponsibilityParentText();
```

Comment:
Å bruke et faktisk tegn her som skille vil nok føre til problemer med skjermleser, jeg har ikke testet det men jeg kan se for meg at den kommer til å være uforventet når den leses opp.